tutorials/tutorials/getting-started.md

```diff
@@ -45,15 +45,16 @@ Ensure you have the following installed:
     main().then(() => process.exit(0))
     ```
 
-3. Download the artifacts in the same folder:
+3. From the root folder run:
     ```
-    wget -c https://artifacts.nevermined.network/421613/public/contracts_v3.5.2.tar.gz -O -| tar -xz --one-top-level=./artifacts
+    mkdir artifacts && cd artifacts && wget -c https://artifacts.nevermined.network/421613/public/contracts_v3.5.2.tar.gz && tar -xzvf contracts_v3.5.2.tar.gz
     ```
+    This will create an "artifacts" folder and download in it the Nevermined smart contract artifacts.
 
 ## Running the Code
 Execute the following command to run the TypeScript file:
 
-    npm install @nevermined-io/sdk
+    yarn start
```

Comment:
I don't think this makes sense because we don't know how the user is using the code. This would only work on the `getting-started` code folder. I would just remove this part